curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6148e41c728f65138addf9cc.md

```diff
@@ -7,14 +7,24 @@ dashedName: step-66
 
 # --description--
 
-Setting the `scroll-behavior` to `smooth` is preferred by most users. However, some users find this to be too slow, and prefer to have the scrolling happen instantaneously.
+Certain types of motion-based animations can cause discomfort for some users. In particular, people with <dfn>vestibular</dfn> disorders have sensitivity to certain motion triggers.
 
-There exists a media rule to set CSS based on the user's browser settings. This media rule is called `prefers-reduced-motion`, and expects one of the following values:
+The `@media` at-rule has a media feature called `prefers-reduced-motion` to set CSS based on the user's preferences. It can take one of the following values:
```

Comment:
```suggestion
The `@media` at-rule has a media condition called `prefers-reduced-motion` to set CSS based on the user's preferences. It can take one of the following values:
```

Reply:
The correct technical term is [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media#media_features).

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

Reply:
True, I wanna to use condition, because it was similar to this article. Waking up, I noticed it is talking about different concept 🤣
 https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/